gradle/taskProperties.kts

```diff
@@ -0,0 +1,14 @@
+gradle.projectsEvaluated {
```

Comment:
:x: This is Groovy :)
Because the filename does not match `.gradle.kts` Gradle falls back to the Groovy DSL.

Reply:
Doh! User mistake. I feel like we should document that in some form. I'd personally prefer if we'd fail the build if the file extension is not something we recognize. Probably the same thing would happen if you have a typo.